src/default.nix

```diff
@@ -317,9 +316,7 @@ in let
             allPackages;
 
           defaultPackage =
-            allPackages
-            ."${dreamLockInterface.defaultPackageName}"
-            ."${dreamLockInterface.defaultPackageVersion}";
+            allPackages."${dreamLockInterface.defaultPackageName}"."${dreamLockInterface.defaultPackageVersion}";
```

Comment:
This change violates column 80. Please revert.
Upstream issue: https://github.com/kamadorueda/alejandra/issues/331

Reply:
done